src/audio/tdfb/tune/sof_bf_filenames_helper.m

```diff
@@ -53,8 +53,8 @@
 bf.sofctl3_fn = fullfile(bf.sofctl3_path, sprintf('coef_%s.txt', idstr));
 bf.tplg1_fn = fullfile(bf.tplg1_path, sprintf('coef_%s.m4', idstr));
 bf.sofctl4_fn = fullfile(bf.sofctl4_path, sprintf('%s.txt', idstr));
-bf.ucmbin3_fn = fullfile(bf.sofctl3_path, sprintf('%s.bin', idstr));
-bf.ucmbin4_fn = fullfile(bf.sofctl4_path, sprintf('%s.bin', idstr));
+bf.ucmbin3_fn = fullfile(bf.sofctl3_path, sprintf('%s.blob', idstr));
```

Comment:
Why the rename ?

Reply:
I started with UCM with .blob and thought that maybe best to have the same for all blobs instead of .bin. Would .bin be better?  I have no idea really but unifying them should be good.

Reply:
yes, pls do unify. ```.bin``` is more obvious its a binary.

Reply:
Yep. I dropped the commit for rename, and will change the few blobs in UCM side instead.